tsconfig.json

```diff
@@ -18,8 +18,8 @@
     "core/templates/dev/head/mathjaxConfig.ts",
     "core/templates/dev/head/domain/classifier/PredictionResultObjectFactory.ts",
     "core/templates/dev/head/expressions/ExpressionParserService.js",
-    "core/templates/dev/head/filters/ConvertToPlainTextFilter.ts",
-    "core/templates/dev/head/filters/TruncateAtFirstLineFilter.ts"
+    "core/templates/dev/head/filters/string-utility-filters/convert-to-plain-text.filter.ts",
```

Comment:
@ankita240796 I have a question -- why are these treated specially? I.e. why are they in tsconfig.json when other files/filters are not?

Reply:
Hi @seanlip, the answer to the question is [here](https://github.com/oppia/oppia/pull/6324#discussion_r284653472). Since we still do not have webpack in extensions folder some files are still included through scripts tags and need to be compiled separately.

Reply:
Oh I see, thanks. Would it be possible to add that here as a note/TODO in a subsequent PR? Otherwise no one reading this file will have any idea...

Reply:
Sure, I will issue a PR adding a note by tomorrow. Is that ok?

Reply:
Yes, thanks @ankita240796!